### PR TITLE
Just some suggestions

### PR DESCRIPTION
**Pull request by felipe1234-dev**
It would be cool to be able to add custom styles and a className for the loader wrapper as well.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -122,7 +126,7 @@ export default class ExampleComponent extends React.PureComponent {
 
     return (
       <div
-        className={!startFadeOut ? this.state.wrapperStyles : fadeOutWrapperStyles}
+        className={`${!startFadeOut ? this.state.wrapperStyles : fadeOutWrapperStyles}${!!wrapperClassName ? wrapperClassName : ""}`}
```
Comment (regrunge) [inline src/index.js]: There should be a space between the two classes

File: src/index.js [hunk 2]
```diff
@@ -14,6 +14,8 @@ export default class ExampleComponent extends React.PureComponent {
     loaderSize: PropTypes.string,
     color: PropTypes.string,
     textStyles: PropTypes.object,
+    wrapperStyles: PropTypes.object,
```
Comment (regrunge) [inline src/index.js:17]: Could you add these 2 props to the readme file and a short explanation? Thanks

Comment (regrunge) [reply src/index.js:17]: @felipe1234-dev

Comment (felipe1234-dev) [reply src/index.js:17]: Really sorry for taking so long to reply. I was very busy with my project (that uses yours) and I just forgot about everything else. So, yep, I'm gonna do that.

**Pull request merged**